src/style.css

```diff
@@ -0,0 +1,22 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.conteiner {
+  display: grid;
+  grid-template-rows: repeat(3, 100px);
+  grid-template-columns: 100%;
+}
+
+.block:nth-child(1) {
```

Comment:
it is ok for now, but will be better to use specific classes(block-red `class="block block-red" if you also need general class`) with separate class names you can reuse styles, but sometimes it is ok to use  nth-child